core/edit_text_file.py

```diff
@@ -16,38 +15,29 @@
 )
 
 _edit_files = {
-    "additional words": os.path.join(
-        REPO_DIR, "core", "vocabulary", "vocabulary.talon-list"
-    ),
-    "alphabet": os.path.join(REPO_DIR, "core", "keys", "letter.talon-list"),
-    "homophones": os.path.join(REPO_DIR, "core", "homophones", "homophones.csv"),
-    "search engines": os.path.join(
-        REPO_DIR, "core", "websites_and_search_engines", "search_engine.talon-list"
-    ),
-    "unix utilities": os.path.join(
-        REPO_DIR, "tags", "terminal", "unix_utility.talon-list"
-    ),
-    "websites": os.path.join(
-        REPO_DIR, "core", "websites_and_search_engines", "website.talon-list"
-    ),
+    "additional words": "core/vocabulary/vocabulary.talon-list",
```

Comment:
I wonder if we should just move this stuff to a talon list while we're in here?

Reply:
done!